qbicsoftware/data-manager-app
# Enable User to view Raw Data for their measurements

**What was changed**
1.) Introduced a dedicated view which will enable a user to view the data generated for their measurements
2.) Provide the correct empty state disclaimers if no samples/measurment were registered
3.) Add Url Generation and download for data download:

**How to test** 
Provide values for the environment variables employed to handle the openbis connection containing the data:

"openbis.datasource.as.url=${OPENBIS_ASS_URL:}"
"openbis.datasource.dss.url=${OPENBIS_DSS_URL:}"
"server.download.api.measurement.url=${DOWNLOAD_MEASUREMENT_API_URL:}"

1.) Go to Project 007 and switch to enabled raw data tab and observe raw data information!
2.) Try to download files with generated urls for raw data
